### PR TITLE
Jax cna and expression loading

### PR DESCRIPTION
**Pull request by sprintell**

### CODE REVIEW

**Comment by Artenides** [comment]
In the TransformerService you have this row:
!treatment.getCurrentDrug().equals(null)){
I think this shouldn't be an equals call, but a ==

**Comment by CsabaHalmagyi** [comment]
In the TransformerService you have this:
!treatment.getCurrentDrug().equals(null)){

I think for nullcheck == should be used.

File: data-model/src/main/java/org/pdxfinder/rdbms/repositories/MappingEntityRepository.java [hunk 1]
```diff
@@ -63,13 +63,7 @@
 
 
     /*
```
Comment (rmwthorne) [inline data-model/src/main/java/org/pdxfinder/rdbms/repositories/MappingEntityRepository.java]: This comment can be deleted

File: data-services/src/main/java/org/pdxfinder/services/MappingService.java [hunk 2]
```diff
@@ -8,11 +8,11 @@
 import org.neo4j.ogm.json.JSONArray;
 import org.neo4j.ogm.json.JSONException;
 import org.neo4j.ogm.json.JSONObject;
-import org.pdxfinder.admin.pojos.MappingContainer;
+import org.pdxfinder.services.mapping.MappingContainer;
 import org.pdxfinder.graph.dao.OntologyTerm;
 import org.pdxfinder.graph.repositories.OntologyTermRepository;
 import org.pdxfinder.rdbms.dao.MappingEntity;
-import org.pdxfinder.admin.zooma.*;
+import org.pdxfinder.services.zooma.*;
```
Comment (rmwthorne) [inline data-services/src/main/java/org/pdxfinder/services/MappingService.java]: Avoid wildcard imports

File: data-services/src/test/java/org/pdxfinder/services/MappingServiceTest.java [hunk 3]
```diff
@@ -27,11 +28,13 @@
     private MappingEntityRepository mappingEntityRepository;
 
     @Test
+    @Ignore
```
Comment (rmwthorne) [inline data-services/src/test/java/org/pdxfinder/services/MappingServiceTest.java]: This test should be fixed or removed

File: indexer/src/main/java/org/pdxfinder/dataloaders/UniversalLoaderOmic.java [hunk 4]
```diff
@@ -42,6 +42,8 @@ public void loadOmicData(ModelCreation modelCreation, Group providerGroup, Strin
 
         reportManager = (ReportManager) context.getBean("ReportManager");
 
+        log.info("Loading {} data for model {} ", dataType, modelCreation.getSourcePdxId());
```
Comment (rmwthorne) [inline indexer/src/main/java/org/pdxfinder/dataloaders/UniversalLoaderOmic.java]:
```suggestion
        log.trace("Loading {} data for model {} ", dataType, modelCreation.getSourcePdxId());
```

File: indexer/src/main/java/org/pdxfinder/postload/CreateDataProjections.java [hunk 5]
```diff
@@ -83,6 +83,10 @@
     //"treatment name"=>"set of model ids"
     private Map<String, Set<Long>> patientTreatmentDP = new HashMap<>();
 
+
+    //"transcriptomics"=>set of model ids
```
Comment (rmwthorne) [inline indexer/src/main/java/org/pdxfinder/postload/CreateDataProjections.java]: Comment here is unnecessary

File: indexer/src/main/java/org/pdxfinder/postload/CreateDataProjections.java [hunk 6]
```diff
@@ -351,22 +357,22 @@ private void createCNADataProjection(){
...
+                    copyNumberAlterationDP.put(m.getHgncSymbol(), newSet);
+                }
             }
 
-            count++;
-            if(count%100 == 0) {log.info("Processed "+count+" CNA molchar objects");
+        }
+
+        count++;
+        if(count%100 == 0) {log.info("Processed "+count+" CNA molchar objects");
```
Comment (rmwthorne) [inline indexer/src/main/java/org/pdxfinder/postload/CreateDataProjections.java]:
```suggestion
        if(count%100 == 0) {log.trace("Processed "+count+" CNA molchar objects");
```

File: indexer/src/main/java/org/pdxfinder/postload/CreateDataProjections.java [hunk 7]
```diff
@@ -377,6 +383,43 @@ private void createCNADataProjection(){
...
+                    Set<Long> newSet = new HashSet<>();
+                    newSet.add(modelId);
+                    transcriptomicsDP.put(m.getHgncSymbol(), newSet);
+                }
+            }
+
+        }
+
+        count++;
+        if(count%100 == 0) {log.info("Processed "+count+" Transcriptomics molchar objects");
```
Comment (rmwthorne) [inline indexer/src/main/java/org/pdxfinder/postload/CreateDataProjections.java]: Remove as per https://github.com/PDXFinder/pdxfinder/pull/95/files#r330076068

File: indexer/src/main/java/org/pdxfinder/postload/CreateDataProjections.java [hunk 8]
```diff
@@ -799,41 +843,50 @@ private void createModelForQueryDataProjection(){
 
                         for(MolecularCharacterization molc : s.getMolecularCharacterizations()){
 
-                                if (molc.getPlatform() != null && !molc.getType().isEmpty()) {
-                                    String platformName = molc.getPlatform().getName();
+                            if (molc.getPlatform() != null && !molc.getType().isEmpty()) {
+                                String platformName = molc.getPlatform().getName();
+
+                                //if (dataImportService.countMarkerAssociationBySourcePdxId(mc.getSourcePdxId(), mc.getDataSource(), platformName) > 0) {
```
Comment (rmwthorne) [inline indexer/src/main/java/org/pdxfinder/postload/CreateDataProjections.java]: This comment is unnecessary

File: indexer/src/main/java/org/pdxfinder/utils/CreateLocalFeeds.java [hunk 9]
```diff
@@ -103,15 +118,15 @@ public void run(String... args) throws Exception {
 
             createJAXFeeds();
 
-            createIRCCFeeds();
+          //  createIRCCFeeds();
```
Comment (rmwthorne) [inline indexer/src/main/java/org/pdxfinder/utils/CreateLocalFeeds.java]: This commented out code should be removed

File: data-model/src/main/java/org/pdxfinder/rdbms/repositories/MappingEntityRepository.java [hunk 10]
```diff
@@ -62,25 +62,7 @@
 
 
 
-    /*
```
Comment (rmwthorne) [inline data-model/src/main/java/org/pdxfinder/rdbms/repositories/MappingEntityRepository.java]: This comment should be deleted

File: indexer/src/main/java/org/pdxfinder/utils/CreateLocalFeeds.java (hunk 9 above)
Comment (sprintell) [reply indexer/src/main/java/org/pdxfinder/utils/CreateLocalFeeds.java]: https://github.com/PDXFinder/pdxfinder/commit/7002eb76912fbef17346a18365842855d1d5efae

**Pull request merged**